### PR TITLE
Add view that redirects to the file serving view for easier code samples.

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

**Comment by darkwing** [comment]
Don't kill me but it's "attachment", not "attachement", in `full_attachement_url`

File: kuma/wiki/tests/test_views.py [hunk 1]
```diff
@@ -3231,6 +3234,63 @@ def test_code_sample_iframe_embed(self):
...
+        eq_(response.status_code, 302)
+
+        # then build the document and revision we need to test
+        attachment = Attachment.objects.get(title='An uploaded file')
+        filename = attachment.current_revision.filename()
+        url_css = 'url("files/%(attachment_id)s/%(filename)s")' % {
+            'attachment_id': attachment.id,
+            'filename': filename,
+        }
+        d, r = doc_rev("""
```
Comment (groovecoder) [inline kuma/wiki/tests/test_views.py]: `d, r` should be `doc, rev` ?

**Comment by groovecoder** [comment]
This looks good for going forward, but it doesn't stop the `SelectorSyntaxError: Expected selector, got <DELIM '/' at 7>` until all the code samples are updated. Are we willing to keep getting all those errors until all the content is cleaned up?

**Comment by jezdez** [comment]
@groovecoder Yeah, this is the correct fix and I don't mind a few error that push the writers to cleanup. I'd like to say 'hey, let's write a data migration to go through all code sample documents and fix it automatically' but well, the writers have more manpower and they may want to do that manually anyway.

**Comment by jezdez** [comment]
@darkwing ZOMG gonna call you eagle eye walsh from now on, good catch!

**Comment by jezdez** [comment]
@darkwing Oh also, I obviously wanted to pronounce the word with a French accent ``attachément`` :fr:

**Comment by darkwing** [comment]
@jezdez I know how much you wacky Europeans like extra vowels... :D

**Comment by jezdez** [comment]
@groovecoder So @a2sheppy says it's okay that they fix the bugs

**Comment by a2sheppy** [comment]
Note that Jannis agrees to let the writing community know when. This lands so we can watch for problems.

Jannis also suggested mature groovecoder could help find at least some places needing fixes with some clever SQL queries...

Eric Shepherd

**Comment by groovecoder** [comment]
@a2sheppy - I just merged and pushed this.

**Comment by groovecoder** [comment]
@hoosteeno - can we punt the SQL query to you? Basically, need to search for all the docs that have code samples with "url('some-attachment-resource.ext')" in them

**Comment by hoosteeno** [comment]
I'll dig in and reach out in IRC with questions.

**Comment by groovecoder** [comment]
@hoosteeno - you can also get a good idea of which pages have samples with broken values from the New Relic error reports.

https://rpm.newrelic.com/accounts/263620/applications/3172075/traced_errors

**Pull request merged**